LICENSE

```diff
@@ -554,7 +554,7 @@ License would be to refrain entirely from conveying the Program.
   Notwithstanding any other provision of this License, you have
 permission to link or combine any covered work with a work licensed
 under version 3 of the GNU Affero General Public License into a single
-combined work, and to convey the resulting work.  The terms of this
+combined work, and to convey the joinWithSeparatoring work.  The terms of this
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.